javascript/modules/client/src/internal/utils/client.ts

```diff
@@ -0,0 +1,48 @@
+import { DaoMetadata } from "../interfaces/client";
+
+// TODO: delete this file
+
+export function getDummyDao(addressOrEns?: string): DaoMetadata {
+  const dummyDaoNames = [
+    "Patito Dao",
+    "One World Dao",
+    "Sparta Dao",
+    "Yggdrasil Unite",
+  ];
+  const dummyDaoEns = [
```

Comment:
Temporary code related to mock data, I would personally put it on a file clearly named as such, like `temp-mock.ts`  or similar. Otherwise, visibility of that is much lower.